dev.env

```diff
@@ -29,3 +30,6 @@ export GLEANER_GRAPH=${GLEANER_BASE}/datavol/graph
 
 # Kubernetes deployment helper
 export KUBECONFIG=~/.kube/config:~/polder-federated-search/helm/polder.config
+
+# Google Analytics property ID
+export GOOGLE_ANALYTICS_ID=
```

Comment:
After this change gets merged, you'll need to add this line in your `.env` for Google events to show up from your development machine.